Transport/Simple.H

```diff
@@ -405,15 +428,16 @@ struct SimpleTransport
 
           amrex::Real muloc, xiloc, lamloc;
           amrex::Real Ddiag[NUM_SPECIES] = {0.0};
-          transport(
-            wtr_get_xi, wtr_get_mu, wtr_get_lam, wtr_get_Ddiag, T, rho, massloc,
-            Ddiag, muloc, xiloc, lamloc, tparm);
+          amrex::Real chi_mix[NUM_SPECIES] = {0.0};
```

Comment:
we might want to wrap this definition and chi_out with some if conditions based on whether they are asked for. I guess the issue is that our current call doesn't let the user do that. By default it asks for all coeffs. Maybe that needs to change. Thoughts anyone?

Reply:
Agreed, we can just not write to that variable if it is not asked for.  Then the use can pass a dummy scalar in that slot and all will be well.

Reply:
The only way I can see to not get chi would be to grab the parmparse flag in the transport params init and check the tparm here? is that a reasonable solution? currently it is caught under a pelelm.use_soret flag but I could change to transport.use_soret (with some appropriate changes in the pelelmex PR)

Reply:
I kinda like this idea. Happy to hear alternatives from others as well. But I would vote for your suggestion.

Reply:
Either would work, and the current version is fine.  It kinda makes sense to have this under transport-specific flags, but that would require that C, LM and LMeX make a bunch of changes that will break everyone's input files, and that's probably a bad idea.

Reply:
I've added in the call in the transport params init to check if use_soret is flagged and adjusted the wtr_get_chi boolean to check this instead (also addresses the comment above) - let me know if this looks good